<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reglage</class>
 <widget class="QDialog" name="Reglage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background : rgb(77, 86, 86  );
color : rgb(255,255,250);
font-family: &quot;New Century Schoolbook&quot; ;</string>
  </property>
  <widget class="QPushButton" name="Valider_Plaque">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Valider_Plaque{
background: rgb(52, 152, 219);
border-radius:5px;
 }
#Valider_Plaque::hover { 
background : rgb(21, 67, 96 );
color:white;
}
</string>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QLabel" name="Texte_Larg_Plaque">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Largeur de la Plaque (mm)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Largeur_Plaque">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Epaisseur_Plaque">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Longueur_Plaque">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Texte_Epais_Plaque">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Epaisseur de la Plaque (mm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Texte_Long_Plaque">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Longueur  de la Plaque (mm)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
